README.md

```diff
@@ -4,40 +4,85 @@
 
 Document datasets in Statistics Norway
 
-## Usage example
+## Usage
 
-1. Run `from datadoc import main; main("./path/to/your/dataset")` to run Datadoc on a dataset of your choosing.
-1. Complete metadata as you desire
-1. Click `Lagre` to save a metadata document together with your dataset
-
-### If the datadoc package is not installed
+![DataDoc in use](./doc/change-language-example.gif)
 
-1. Clone this repo to your Jupyter instance (or local machine)
-1. Open the `DataDoc.ipynb` Notebook and run the cell to see the example dataset
+### From Jupyter
 
-![DataDoc in use](./doc/change-language-example.gif)
+1. Open <https://jupyter.dapla-staging.ssb.no> or another Jupyter Lab environment
+1. Run `pip install ssb-datadoc` in the terminal
+1. Upload a dataset to your Jupyter server (e.g. <https://github.com/statisticsnorway/datadoc/blob/master/klargjorte_data/person_data_v1.parquet>)
+1. Run `from datadoc import main; main("./person_data_v1.parquet")` in a notebook
+1. Datadoc will open in a new tab
 
 ## Contributing
 
+### Prerequisites
+
+- Python >3.8 (3.10 is preferred)
+- Poetry, install via `curl -sSL https://install.python-poetry.org | python3 -`
+
 ### Dependency Management
 
-Poetry is used for dependency management.
+Poetry is used for dependency management. Upon cloning this project first install necessary dependencies, then run the tests to verify everything is working.
```

Comment:
Poetry is used for dependency management, and [Poe the Poet](https://github.com/nat-n/poethepoet) is used for running `poe` tasks within poetry's virtualenv.